resources/prosody-plugins/util.lib.lua

```diff
@@ -131,13 +131,15 @@ function update_presence_identity(
         "Presence with identity inserted %s", tostring(stanza))
 end
 
--- Utility function to check whether feature is present and enabled. Apply
--- feature restriction, only if features are present in the session(coming from
--- the token) and the value of the feature is not true.
-function apply_features_restriction(session, feature)
-    if (session.jitsi_meet_context_user ~= nil
-        and session.jitsi_meet_context_user["features"] ~= nil
-        and session.jitsi_meet_context_user["features"][feature] ~= "true") then
+-- Utility function to check whether feature is present and enabled. Allow
+-- a feature if there are features present in the session(coming from
+-- the token) and the value of the feature is true.
+-- If features is not present in the token we skip feature detection and allow
+-- everything.
+function is_feature_allowed(session, feature)
+    if (session.jitsi_meet_context_user == nil
+        or session.jitsi_meet_context_user["features"] == nil
```

Comment:
the naming of this looks good (though i think the name usage needs to change elsewhere), but is this logic right?

if there IS a user present, but the `features` map is null, should it be allowed?  i thought it was "if there is a user present, then the explicit feature MUST be present and set to true"

Reply:
No, if there are no features map present allow it. This was my intention. So those using the token without feature can continue using it. And when this is enforced from server only then obey it.

Reply:
Yep I had left the old function name invocation, now it is fixed.